### PR TITLE
Update a Loader test

### PR DESCRIPTION
**Pull request by kouvel**
See https://github.com/dotnet/coreclr/pull/2015
Related to #1187

### CODE REVIEW

**Comment by kouvel** [comment]
@sergiy-k @stephentoub

File: src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs [hunk 1]
```diff
@@ -15,12 +15,18 @@ public class AssemblyLoadContextTest
     {
         private const string TestAssembly = "System.Runtime.Loader.Test.Assembly";
 
+        [ActiveIssue(/* dotnet/coreclr */ 1187, PlatformID.Windows)] // dependency on coreclr behavior, waiting for new coreclr
```
Comment (stephentoub) [inline src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs]: This is failing on Unix as well:
```
08:15:07    System.Runtime.Loader.Tests.AssemblyLoadContextTest.GetAssemblyNameTest_ValidAssembly [FAIL]
08:15:07       System.NotSupportedException : Assembly.Load with a Codebase is not supported on Windows Phone.
08:15:07       Stack Trace:
08:15:07 Discovered:  System.ComponentModel.Annotations.Tests
08:15:07             at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
08:15:07             at System.Runtime.Loader.Tests.AssemblyLoadContextTest.GetAssemblyNameTest_ValidAssembly()
```

Comment (kouvel) [inline src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs]: I'll need to submit that separately, will remove from this PR for now. I tested it locally and it works, the CI is just using an older mscorlib (probably from last successful build).

Comment (kouvel) [inline src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs]: Nevermind, I already did put it in a different PR :). I'll just need to wait for https://github.com/dotnet/coreclr/pull/2015 to be submitted and wait for that to be published before submitting this one.

**Comment by kouvel** [comment]
@dotnet-bot test this please

**Pull request merged**